### PR TITLE
Templating/Migration test renaming and refactoring

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

**Comment by alexandraorth** [comment]
!rtg - cassandra timeouts

**Comment by alexandraorth** [comment]
!rtg - cassandra timeouts

File: grakn-graql/src/test/java/ai/grakn/graql/internal/template/TemplateParserTest.java [hunk 1]
```diff
@@ -128,42 +111,52 @@ public void noSpacesBetweenTokensTest(){
...
                 "for (whale in <whales> ) do {" +
                 "$x isa whale has name <whale>;\n}";
 
-        String expected =
-                "insert $x0 isa whale has name \"shamu\";\n" +
-                        "$x1 isa whale has name \"dory\";";
-
-        Map<String, Object> data = new HashMap<>();
-        data.put("whales", Arrays.asList("shamu", "dory"));
+        String expected = "insert $x0 isa whale has name \"shamu\";\n" +
```
Comment (fppt) [inline grakn-graql/src/test/java/ai/grakn/graql/internal/template/TemplateParserTest.java]: HA. love the names :)

File: grakn-test/src/test/java/ai/grakn/test/migration/MigratorTestUtils.java [hunk 2]
```diff
@@ -123,46 +123,50 @@ public static Resource getResource(GraknGraph graph, Thing thing, TypeLabel labe
...
-        Collection<Entity> pets = graph.getEntityType("pet").instances();
-        assertEquals(9, pets.size());
+    public static void assertPetGraphCorrect(GraknSession session){
+        try(GraknGraph graph = session.open(GraknTxType.READ)) {
+            Collection<Entity> pets = graph.getEntityType("pet").instances();
+            assertEquals(9, pets.size());
 
-        Collection<Entity> cats = graph.getEntityType("cat").instances();
-        assertEquals(2, cats.size());
+            Collection<Entity> cats = graph.getEntityType("cat").instances();
```
Comment (fppt) [inline grakn-test/src/test/java/ai/grakn/test/migration/MigratorTestUtils.java]: Holy mother of tabs!

Comment (alexandraorth) [inline grakn-test/src/test/java/ai/grakn/test/migration/MigratorTestUtils.java]: On my laptop they show up as spaces :( Swear I have not gone back to the dark side

**Pull request merged**